ddl/column.go

```diff
@@ -157,9 +222,8 @@ func (d *ddl) onAddColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 			return ver, errors.Trace(err)
 		}
 
-		// Adjust column offset.
-		d.adjustColumnOffset(tblInfo.Columns, tblInfo.Indices, offset, true)
-		columnInfo.State = model.StatePublic
+		d.adjustColumnOffset(tblInfo.Columns, tblInfo.Indices, 0, lastColOffset, true)
```

Comment:
Mulit-add support add columns with first|after position. Such testcase is in column_change_test.testAddMultipleColumns.
It is ok to split adjustColumnOffset into adjustColumnOffsetForDrop and adjustColumnOffsetForAdd